static/js/modules/filters-event.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+var $ = require('jquery');
+var _ = require('underscore');
+
+var FilterPanel = require('fec-style/js/filter-panel').FilterPanel;
+var URI = require('urijs');
+
+function lineNumberFilters() {
+  this.filterPanel = new FilterPanel();
```

Comment:
Hm that's strange. It's not actually a `form` element, so maybe that's it? Trying something...